packages/rpcs/wallet_importHardwareWalletAccountGroupOrAccount/index.js

```diff
@@ -0,0 +1,148 @@
+import {
+  enums,
+  map,
+  dbid,
+  stringp,
+  mapp,
+  base32UserAddress,
+  ethHexAddress,
+  or,
+  oneOrMore,
+  password,
+} from '@fluent-wallet/spec'
+import {decode, encode} from '@fluent-wallet/base32-address'
+import {encrypt} from 'browser-passworder'
+
+export const NAME = 'wallet_importHardwareWalletAccountGroupOrAccount'
+
+const AddressSchema = [
+  map,
+  {closed: true},
+  ['address', [or, base32UserAddress, ethHexAddress]],
+  ['nickname', stringp],
+]
+
+const BasicSchema = [
```

Comment:
maybe we should add the data-format Schema to the hdPath of various chains?

Reply:
the schema of `accountGroupData` set to any object deliberately here
since we don't know what other hardware wallets expects from fluent

when interacting with hardware wallet:
1. we import `data1` from hardware wallet and init fluent data structure around `data1` as `data2`
2. we ask hardware wallet with `data3` for signature

the `accountGroupData` is intended to be the helper metadata so that we can use get `data3` from user input data (things to sign) and `data2`.